### ISSUE TITLE
Invalid ripple address

### ISSUE THREAD

**Issue by brianlong** [ripple/ripple-client#469]
Bitstamp says "Invalid ripple address" for public address: 

rJsSrfDZ5C4z8FigfKvdqLsF2heWt8Cy1w

How are the Ripple addresses validated? Is mine valid?

Thanks

**Comment by headshot-g** [comment]
I created an account just to comment on this thread.

I have a Bitstamp account and I ran into the same thing when trying to create my white-list. I just did a workaround I thought would work, and it did. Instead of trying to white-list an empty account and failing (because its empty) I just turned the white-list requirement off, then sent 20 XRP to my Ledger Nano S, turned the white-list requirement back on and made a white-list entry. Now that the ledger was funded, it let me create the white-list without any trust issues or prerequisite requirements not being met. No need to requisition help from helpful and generous internet randoms :)

TLDR: Send yourself 20 XRP with white-list turned off on Bitstamp before you turn it on and attempt to create a white-list exception, its only a problem during white-list creation.
